packages/dbadmin/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist",
+    "resolveJsonModule": true,                        /* Enable importing .json files */
```

Comment:
I would expect this one then also in the top-level `tsconfig.json`.

Reply:
you mean the resolveJsonModule ?